### DISCUSSION TITLE
How to install code-server on Ubuntu 18.04 with arm64

### DISCUSSION THREAD

**Discussion by Tumb1eweed** [microsoft/vscode-cpptools#9852] [General]
ldd --version shows 2.27
and I use curl -fsSL https://code-server.dev/install.sh | sh to install code-server and run shows:
code-server: /lib/aarch64-linux-gnu/libc.so.6: version `GLIBC_2.28' not found (required by code-server)
code-server: /lib/aarch64-linux-gnu/libc.so.6: version `GLIBC_2.29' not found (required by code-server)

**Comment by sean-mcmanus** [comment]
I don't know. Our extension doesn't support code-server so you might want to ask somewhere else.
